inference-engine/src/vpu/common/src/ngraph/operations/static_shape_broadcast.cpp

```diff
@@ -87,6 +92,8 @@ bool StaticShapeBroadcast::visit_attributes(ngraph::AttributeVisitor& visitor) {
         mode = "explicit";
     } else if (m_mode.m_type == ngraph::op::BroadcastType::NUMPY) {
         mode = "numpy";
+    } else if (m_mode.m_type == ngraph::op::BroadcastType::BIDIRECTIONAL) {
+        mode = "bidirectional";
     }
```

Comment:
I think it's better to add the final `else` branch here and throw in case when the mode is not `NUMPY`, `BIDIRECTIONAL` or `EXPLICIT`.